### ISSUE TITLE
Portable Fluid Interface doest work?

### ISSUE THREAD

**Issue by Gl1tchAlpha** [xiewuzhiying/VS-Addition#49]
### This issue occurs when only Valkyrien Skies and addons are installed and no other mods

- [X] I have tested this issue and it occurs when no other mods are installed

### Minecraft Version

1.20.1

### Mod Loader

Forge

### Issue description

i saw the working mode gui and i keep testing it but it doesn't work
(the checkmark box is a lie i have alexs caves, create addons, cuffed, vs tournament, security craft)

### Issue reproduction

change the working mod
put the 2 interfaces together
nothing happens

**Comment by MulatramAwA** [comment]
作者已多次声明工作模式并不可用！

**Comment by xiewuzhiying** [comment]
No, it works, but you still need a contraption to relay. I've been remaking some old stuff lately, maybe I'll remake it.

**Comment by ZhouDreams** [comment]
根据我和朋友们在基友服的实践，目前版本下需要发送方和接收方至少一边是旋转的动态结构。比方说你想要将地上的熔岩输入船上的储罐，需要建一个旋转的动态结构（如动力轴承）作为中介，旋转结构上一边接地面，另一边接船上。此外要注意设置接口的工作模式。

**Comment by Gl1tchAlpha** [comment]
> No, it works, but you still need a contraption to relay. I've been remaking some old stuff lately, maybe I'll remake it.

oh ok thx i thought you put one on the ground and the other on the ship thx tho

**Comment by ThunderV01d** [comment]
Is there any update on this issue? Still experiencing it right now.
